core/client/app/templates/components/gh-editor-save-button.hbs

```diff
@@ -1,13 +1,11 @@
-<button type="button" {{action "save"}} class="btn btn-sm js-publish-button {{if isDangerous 'btn-red' 'btn-blue'}}">
-    {{saveText}}
-</button>
+{{gh-spin-button type="button" class=buttonClass action="save" buttonText=saveText}}
 
 {{#gh-dropdown-button dropdownName="post-save-menu" classNameBindings=":btn :btn-sm isDangerous:btn-red:btn-blue btnopen:active :dropdown-toggle :up"}}
     <i class="options icon-arrow2"></i>
     <span class="sr-only">Toggle Settings Menu</span>
 {{/gh-dropdown-button}}
 {{#gh-dropdown name="post-save-menu" closeOnClick="true" tagName="div" classNames="dropdown editor-options"}}
-    <ul class="dropdown-menu dropdown-triangle-bottom-right">
+    <ul class="dropdown-menu pull-right dropdown-triangle-bottom-right">
```

Comment:
@JohnONolan I'm not sure if this was intentional or not (mobile perhaps?), but this dropdown looked off, so I added the pull-right class.